### ISSUE TITLE
[Form Provider Refactor] WorkspaceInviteMessagePage

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#31564]
Coming from https://github.com/Expensify/App/issues/25397#issuecomment-1743272245, we created a new Form provider that works with functional components and we need to refactor all usages of the old Form component.

This issue is specific to refactoring `src/pages/workspace/WorkspaceInviteMessagePage.js`

cc @kowczarz @cdOut

Note: this component also needs to be migrated to a functional component!

**Comment by luacmartins** [comment]
@cdOut @kowczarz are either of you available to work on this one?

**Comment by cdOut** [comment]
Yes! I'm Tymoteusz Kałuzieński from Software Mansion, the expert agency and I would like to work on this task.

**Comment by cooldev900** [comment]
I would like to work on this task.

**Comment by luacmartins** [comment]
Thanks for the interested @cooldev900! However, we already assigned a contributor to work on it.

**Comment by luacmartins** [comment]
Waiting on draft PR

**Comment by luacmartins** [comment]
We have a draft PR

**Comment by c3024** [comment]
Melvin isn't removing the Overdue label. PR is in draft.

**Comment by c3024** [comment]
The issue was already assigned, Melvin. PR is in draft.

**Comment by c3024** [comment]
PR in draft. Functional component migration part is pending.

**Comment by c3024** [comment]
PR in draft. Functional component migration part is under the works.

**Comment by cdOut** [comment]
Currently working on pushing the PR out of draft, Kamil is reviewing the changes I implemented atm.

**Comment by cdOut** [comment]
Working on it, should be pushed out of draft today.

**Comment by luacmartins** [comment]
still working on the draft

**Comment by c3024** [comment]
BZ checklist is not applicable for this as this is form refactor and functional component migration.

**Comment by miljakljajic** [comment]
Thank you for waiting over the holiday season! Sharing a payment summary:

@c3024 is owed 125 USD for their review work on this issue. Please request payment manually in NewDot.

**Comment by c3024** [comment]
@miljakljajic Thanks but I am not eligible for manual requests on NewDot yet.

**Comment by c3024** [comment]
@miljakljajic I can get paid only on Upwork for now.

**Comment by c3024** [comment]
@miljakljajic 

Could you please check [this](https://github.com/Expensify/App/issues/31564#issuecomment-1875208667)?

I am not eligible for manual requests on New Dot yet. I need to be paid on Upwork.
